reacnetgenerator/_path.py

```
@@ -197,55 +202,71 @@ def collect(self):
-                    conflict_atom, conflict_step = np.nonzero(
-                        atomeach[atom[:, None], index]
-                    )
-                    conflict[atom[conflict_atom], index[conflict_step]] = 1
-                    atomeach[atom[:, None], index] = i
-        return atomeach, conflict
+                for molecule_id, (linehz, linetz) in enumerate(
+                    tqdm(
+                        zip(
+                            read_compressed_block(fh),
+                            itertools.zip_longest(*[read_compressed_block(ft)] * 4),
+                        ),
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)